cosmosdb/management_endpoint.bal

```diff
@@ -178,15 +187,15 @@ public client class ManagementClient {
         return mapJsonToContainerType(jsonResponse);
     }
 
-    # Create a container only if the specified container ID does not exist already.
+    # Creates a container only if the specified container ID does not exist already.
     #
     # + databaseId - ID of the database to which the container belongs to
     # + containerId - ID of the new container
-    # + partitionKey - A record of type `PartitionKey`
-    # + indexingPolicy - Optional. A record of type `IndexingPolicy`.
-    # + throughputOption - Optional. Throughput parameter of type int or json.
-    # + return - If successful, returns `Container` if a new container is created or `nil` if container already exists.
-    #            Else returns `Error`.
+    # + partitionKey - A record of `cosmos_db:PartitionKey`
+    # + indexingPolicy - A record of `cosmos_db:IndexingPolicy`.
+    # + throughputOption - Throughput parameter of type int or json.
```

Comment:
Suggestion: "The throughput parameter of `int` or `json` type"

Replace the article "The' at the start of the sentence with "A" if relevant.